setup.py

```diff
@@ -1,8 +1,44 @@
 from setuptools import setup, find_packages
+import os
+import sys
+
+DATA_FILES = []
+CURRENT_DIR = os.getcwd()
+
+# Install icons and desktop files, not sure if /usr/local/share is constant
+if sys.platform == 'linux' and os.environ.get("DISABLE_ICONS_DESKTOP", 0) == 0:
+    if 'bdist_wheel' in sys.argv:
+        sys.exit("Building wheels is disabled. This is not a bug")
+
+    if os.geteuid() == 0:
```

Comment:
Not sure what this is good for.

Reply:
This is because when installing from source `pip install .` pip tries to build a wheel for Vorta using `bdist_wheel`. When downloading from a pypi release it should not be an issue because the package is built as `sdist`.

Reply:
I've meant: if os.geteuid() == 0:
:)

Reply:
That is to check if the user is installing it as root, and sets the local variable appropriately, since the default install directory when running pip as root is /usr/local/